### PR TITLE
Refactor all request:parameter tags

### PR DESCRIPTION
**Pull request by isears**

### CODE REVIEW

File: omod/src/main/java/org/openmrs/web/taglib/functions/Util.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	
+	/**
+	 * This method will make untrusted strings safe for use as JavaScript integers
+	 *
+	 * @param s
+	 * @return either the original value (if an int) or 'null'
+	 */
+	public static String getSafeJsInt(String s){
+		for(int i = 0; i < s.length(); i++){
+			if(!Character.isDigit(s.charAt(i))){
```
Comment (dkayiwa) [inline omod/src/main/java/org/openmrs/web/taglib/functions/Util.java]: How about using something like org.apache.commons.lang.math.NumberUtils.isNumber()

File: omod/src/main/java/org/openmrs/web/taglib/functions/Util.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	}
+	
+	/**
+	 * This method will make untrusted strings safe for use as JavaScript booleans
+	 *
+	 * @param s
+	 * @return one of 'true', 'false', or 'null'
+	 */
+	public static String getSafeJsBoolean(String s){
+		if(s.equals("true") || s.equals("false") || s.equals("0") || s.equals("1")){
```
Comment (dkayiwa) [inline omod/src/main/java/org/openmrs/web/taglib/functions/Util.java]: How about BooleanUtils.toBooleanObject()?

Comment (isears) [inline omod/src/main/java/org/openmrs/web/taglib/functions/Util.java]: Yeah that would be better. I will submit changes shortly (for both functions).

**Comment by dkayiwa** [comment]
Thanks @isears 👍

**Pull request merged**